deposit/src/test/java/edu/unc/lib/deposit/normalize/Simple2N3BagJobTest.java

```diff
@@ -101,18 +99,18 @@ public void depositContainerWithMODSTest() throws Exception {
 		copyTestPackage("src/test/resources/simpleMods.xml", "mods.xml", job);
```

Comment:
Geez there was a lot of stuff related to making containers this way, but i'm fairly certain its happening via the DigitalObjectManager since that is what the AddContainerController is using.